api/src/test/java/com/googlecodesamples/cloud/jss/lds/controller/FileControllerTest.java

```diff
@@ -0,0 +1,94 @@
+package com.googlecodesamples.cloud.jss.lds.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.googlecodesamples.cloud.jss.lds.model.BaseFile;
+import com.googlecodesamples.cloud.jss.lds.model.BaseFileTest;
+import com.googlecodesamples.cloud.jss.lds.service.FileService;
+import org.junit.Assert;
```

Comment:
nit: We typically use the `com.google.common.truth` library for asserts.

See JavaDoc:
https://truth.dev/api/latest/index.html?overview-summary.html

Examples of usage:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/speech/src/test/java/com/example/speech/AdaptationInlinePhraseSetV2IT.java#L19